src.cmake

```diff
@@ -143,8 +143,44 @@ set(GLSLSOURCELIST
     ${ENGINE_DIR}/renderer/glsl_source/skybox_fp.glsl
 )
 
+if (BUILD_CLIENT)
```

Comment:
Complex logic shouldn't go in `src.cmake`. Just the file lists

Reply:
But then we have a chicken-and-egg problem because the list of GLSL files from the list should be known for generating the ` shaders.cpp` file listing them, and then only the path to that file is known to be set in that list.

Reply:
Well, that's annoying to not rely on a variable, but OK, I now rewrote by copy-pasting the name.